### DISCUSSION TITLE
Class/var name for on focus outline

### DISCUSSION THREAD

**Discussion by vnugent** [saadeghi/daisyui#1157] [Q&A] [answered]
How do I customize component [outline](https://tailwindcss.com/docs/outline-width) in tailwind.config.js?  

<img width="369" alt="Screen Shot 2022-09-19 at 5 18 02 PM" src="https://user-images.githubusercontent.com/3805254/191052800-ea45b55a-ac47-4d5b-b5d0-b8f2e5284288.png">

**Comment by saadeghi** [comment, marked as answer]
Examples: 
```html
<input class="input input-bordered m-4 focus:outline-8">

<input class="input input-bordered m-4 focus:outline-primary">

<input class="input input-bordered m-4 focus:outline-dotted">

<input class="input input-bordered m-4 focus:outline-offset-8">
```
https://play.tailwindcss.com/ZL061JkSnh

**Reply by vnugent** [reply]
Thank you.  I was hoping for a global variable that can be applied to button and input.

**Reply by saadeghi** [reply]
Tailwind CSS provides utility classes for the customization so I don't think adding CSS variables would count as an improvement for developer experience 🤔

**Reply by vnugent** [reply]
Perhaps it depends how we view the role of `outline`.  In my mind it's a global preference similar to `--rounded-box` and `--rounded-button`.  For example, I'd want all my buttons and form controls to have a lighter outline.  Right now I think the outline is inherited from the component color which is quite heavy for my taste.
